### ISSUE TITLE
Exception after opening latte file from Tracy Exception screen

### ISSUE THREAD

**Issue by mildabre** [mesour-intellij-plugins/latte-pro-issues#108]
**Describe the bug**

After error in latte template I have clicked on link to template file in Tracy Exception screen and choose option Open in project, after than plugin exception occurs.

**Environment (please complete the following information):**
 - PhpStorm version [e.g. 2023.1.1]
 - Latte Pro [e.g. 1.0.11]

**Code**

```
java.lang.UnsupportedOperationException: Smart mode is not available when LightEdit is active
	at com.intellij.ide.lightEdit.project.LightEditDumbService.reportUnavailable(LightEditDumbService.java:102)
	at com.intellij.ide.lightEdit.project.LightEditDumbService.runWhenSmart(LightEditDumbService.java:26)
	at com.mesour.intellij.latte.configuration.c.f.a(SourceFile:34)
	at com.mesour.intellij.latte.configuration.c.f.a(SourceFile:13)
	at com.mesour.intellij.latte.configuration.c.a.b(SourceFile:69)
	at com.mesour.intellij.latte.configuration.c.a.a(SourceFile:56)
	at com.mesour.intellij.latte.configuration.c.a.<init>(SourceFile:46)
	at com.mesour.intellij.latte.configuration.c.a$a.a(SourceFile:218)
	at com.mesour.intellij.latte.configuration.c.a$a.a(SourceFile:213)
	at com.mesour.intellij.latte.h.d.projectOpened(SourceFile:12)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:649)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:629)
	at com.intellij.util.messages.impl.MessageBusImplKt.executeOrAddToQueue(MessageBusImpl.kt:470)
	at com.intellij.util.messages.impl.ToDirectChildrenMessagePublisher.publish$intellij_platform_core(CompositeMessageBus.kt:259)
	at com.intellij.util.messages.impl.MessagePublisher.invoke(MessageBusImpl.kt:425)
	at jdk.proxy2/jdk.proxy2.$Proxy58.projectOpened(Unknown Source)
	at com.intellij.ide.lightEdit.project.LightEditProjectManager.lambda$fireProjectOpened$0(LightEditProjectManager.java:59)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.openapi.application.impl.ApplicationImpl$3.run(ApplicationImpl.java:513)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:75)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:118)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:42)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:779)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:749)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:918)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:766)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:450)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:791)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:449)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:624)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:447)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:493)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

```

**Comment by mildabre** [comment]
Hi,

today I have reproduce the warning in latte - {$nonexisting} and the link from Tracy exception screen to latte template works excellently. So it looks like that the source of the bug has disappear - but I have done no upgrade IDE or plugin. You can check this also and if everything works we can close this issue.

**Comment by mesour** [comment]
Fixed in 1.2.1
